build.cake

```diff
@@ -0,0 +1,108 @@
+#addin "Cake.FileHelpers"
+
+var target = Argument("target", "Default");
+var project = Argument("project", "*");
```

Comment:
On my Mac, I got an error that this variable is shadowed, that is another variable is defined with the same name. To fix this, you could rename this variable to something like `projectFilter`.